src/App.js

```diff
@@ -23,6 +24,7 @@ function App() {
         <Switch>
           <Route path="/" exact>
             <Searchbar />
+            <PlaceCreateForm />
```

Comment:
I propose to put it in a separate item in the menu as 'add new place' with new route